rpc/params/doc.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package params contains the definition of the request and response parameters for the Juju API Server and
+// Juju API callers (which includes the user-facing clients but also the agents and the controller itself). It includes:
```

Comment:
suggestion: the controller is one type of agent.
```suggestion
// Juju API callers (which includes the user-facing clients and the different agents: controller, machine, and unit). It includes:
```